### PR TITLE
Added sinanju WK

### PR DESCRIPTION
**Pull request by mrnoisytiger**
## Description

Added Sinanju WK PCB

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation
 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Comment by mrnoisytiger** [comment]
Bump 😊

File: keyboards/fjlabs/sinanjuwk/info.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
+{
+    "keyboard_name": "Sinanju WK",
+    "url": "https://www.fjlaboratories.com",
+    "maintainer": "FJLabs",
```
Comment (fauxpark) [inline keyboards/fjlabs/sinanjuwk/info.json:4]:
```suggestion
    "keyboard_name": "Sinanju WK",
    "manufacturer": "FJLabs",
    "url": "https://www.fjlaboratories.com",
    "maintainer": "FJLabs",
    "usb": {
        "vid": "0x7074",
        "pid": "0x1023",
        "device_version": "0.0.1"
    },
```

File: keyboards/fjlabs/sinanjuwk/config.h [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x7074 // FJLabs
+#define PRODUCT_ID   0x1023 // Sinanju WK
+#define DEVICE_VER   0x0001 // Version 1
+#define MANUFACTURER    FJLabs
+#define PRODUCT         Sinanju WK
+
```
Comment (fauxpark) [inline keyboards/fjlabs/sinanjuwk/config.h:28]:
```suggestion
```

File: keyboards/fjlabs/sinanjuwk/readme.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Sinanju WK
+
+The following is the QMK Firmware for the Windz Sinanju WK PCB. 
+ 
+
+* Keyboard Maintainer: FJLabs
+
+* Hardware Supported: Sinanju WK
```
Comment (fauxpark) [inline keyboards/fjlabs/sinanjuwk/readme.md:8]:
```suggestion
* Keyboard Maintainer: FJLabs
* Hardware Supported: Sinanju WK

```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by mrnoisytiger** [comment]
Bump. 😊

**Review by fauxpark — APPROVED** [review]

**Pull request merged**